### ISSUE TITLE
Compatible E-paper panel sourcing resolved>!<

### ISSUE THREAD

**Issue by 229311JK** [sqfmi/Watchy#248]
Compatible 1.54" SPI Monochrome 200x200 Pixel E-paper replacement can be found @the hyperlink below...
   
[https://www.seeedstudio.com/1-54-Monochrome-ePaper-Display-with-200x200-Pixels-p-5776.html](url)

**Comment by DarkZeros** [comment]
Do you think this one is better than the default one, or is it the same? Is it needed to change the code at all?

**Comment by 229311JK** [comment]
> Do you think this one is better than the default one, or is it the same? Is it needed to change the code at all?

Personally, yes & it works great so far; Need to change SQFMI WATCHY code yes.  Your code of course is a different beast...  Thanks 4-sharing, didn't know bout it till now...

**Comment by 229311JK** [comment]
Need a hand w/IDF Arduino core get in touch...

**Comment by 229311JK** [comment]
> Do you think this one is better than the default one, or is it the same? Is it needed to change the code at all?

Did you try-2 get 1-yet?  Noticed a number of them bought up lately.....  Also, please see the above comment too...

**Comment by DarkZeros** [comment]
I didn't buy it since I was checking the documents attached and looked like it is the same controller and display (specs are exactly the same), maybe marketed by another name?

If you need help with something let me know, I am working on a side project to Watchy on my spare time.

**Comment by 229311JK** [comment]
> I didn't buy it since I was checking the documents attached and looked like it is the same controller and display (specs are exactly the same), maybe marketed by another name?
> 
> If you need help with something let me know, I am working on a side project to Watchy on my spare time.

Yep, I spent hours going over datasheets 4-E-paper panels.... This version panel has a different PN & is more responsive than the OEM.  On help, thank you will be in touch....
